src/pages/public_registration/action_accordions/DeletePublication.tsx

```diff
@@ -0,0 +1,160 @@
+  });
+  const handleDelete = async (values: DeleteForm) => {
+    const unpublishRequest = (): UnpublishPublicationRequest => {
+      return selectedDuplicate
+        ? {
+            type: 'UnpublishPublicationRequest',
+            duplicateOf: selectedDuplicate.id,
+            comment: values.deleteMessage,
+          }
+        : { type: 'UnpublishPublicationRequest', comment: values.deleteMessage };
+    };
+
+    setAwaitingUnpublishingResponse(true);
+
+    const unpublishingResponse = await unpublishRegistration(registration.identifier, unpublishRequest());
+    setAwaitingUnpublishingResponse(false);
+    if (isErrorStatus(unpublishingResponse.status)) {
+      dispatch(setNotification({ message: t('feedback.error.update_registration'), variant: 'error' }));
+    }
+    if (isSuccessStatus(unpublishingResponse.status)) {
+      setShowDeleteModal(false);
+      history.push(`/registration/${registration.identifier}?shouldNotRedirect`);
+    }
+  };
+
+  return (
+    <>
+      <Box sx={{ display: 'flex', flexDirection: 'column', gap: '1rem', mt: '1rem' }}>
+        <Divider />
+        <Breadcrumbs
+          sx={{ alignSelf: 'center' }}
+          itemsBeforeCollapse={0}
+          itemsAfterCollapse={0}
+          maxItems={0}
+          aria-label={t('common.show_more_options')}>
+          <Button
+            data-testid={dataTestId.unpublishActions.openUnpublishModalButton}
+            variant="outlined"
+            onClick={() => setShowDeleteModal(true)}>
+            {t('common.delete')}
+          </Button>
+        </Breadcrumbs>
+      </Box>
+      <Modal
+        dataTestId="delete-registration-modal"
+        headingText={t('registration.delete_registration')}
+        open={showDeleteModal}
+        onClose={() => setShowDeleteModal(false)}>
+        <Box sx={{ display: 'flex', flexDirection: 'column', gap: '1rem' }}>
+          <Box>
+            <RequiredDescription />
+          </Box>
+          <Box sx={{ gap: '1rem' }}>
+            <Typography variant="h3">{t('unpublish_actions.unpublish_registration')}</Typography>
+            <Typography>{t('unpublish_actions.unpublish_registration_detail_1')}</Typography>
+            <Typography>{t('unpublish_actions.unpublish_registration_detail_2')}</Typography>
+          </Box>
+
+          <Formik
+            initialValues={{
+              deleteMessage: '',
+            }}
+            validationSchema={deleteValidationSchema}
+            onSubmit={(values, _formikHelpers) => {
+              handleDelete(values);
+            }}>
+            {({ values, errors, touched, handleChange, handleBlur, handleSubmit }) => (
+              <form onSubmit={handleSubmit}>
+                <Box sx={{ my: '1rem' }}>
+                  <Typography variant="h3">{t('registration.is_registration_error_question')}</Typography>
+                  <Typography gutterBottom={true}>{t('unpublish_actions.unpublish_registration_reason')}</Typography>
```

Comment:
kan ta vekk verdien på bolske felter for readability. Om prop er satt -> `true`, ellers -> `false`
```suggestion
                  <Typography gutterBottom>{t('unpublish_actions.unpublish_registration_reason')}</Typography>
```